src/java/com/phenix/pct/PCTCompile.java

```diff
@@ -60,8 +63,10 @@ public PCTCompile() {
 
         fsListId = PCT.nextRandomInt();
         paramsId = PCT.nextRandomInt();
+        compId = PCT.nextRandomInt();
         fsList = new File(System.getProperty("java.io.tmpdir"), "pct_filesets" + fsListId + ".txt"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "java.io.tmpdir" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1192)